share/doc/DEVELOPERS_GUIDE.rst

```diff
@@ -72,7 +72,81 @@ Making a Release
 Similar to the VOTCA containers, releases are also handled by GitHub actions. :code:`votca/votca` has a :code:`release` workflow that can only be triggered manually.
 To trigger it go `here <https://github.com/votca/votca/actions?query=workflow%3Arelease>`_. The release can only be made from the 
 :code:`stable` branch, but testing the creation of a release can be triggered on any branch. To make a release, trigger the action from the
-:code:`stable` branch, pick a new release tag in the :code:`release tag` box (all CHANGELOG files should already contain a section with the tag, but the date will be updated) and type :code:`yesyesyes` into the deploy box. A new release will trigger the creation of the release tag in all involved submodules (plus pull requests for the :code:`stable` to :code:`master` branch, see `below <#updates-from-stable>`__). 
+:code:`stable` branch, pick a new release tag in the :code:`release tag` box (all CHANGELOG files should already contain a section with the tag, but the date will be updated) and type :code:`yesyesyes` into the deploy box. A new release will trigger the creation of the release tag in all involved submodules (plus pull requests for the :code:`stable` to :code:`master` branch, see `below <#updates-from-stable>`__).
+
+Major releases
+~~~~~~~~~~~~~~
+
+In preparation for a major (not minor!) release the following additional steps need to be done:
+-  Create a new branch from the master branch of the :code:votca/votca repository and also in each of the submodules, e.g. :code:stable_bump. 
+   ::
+
+       git checkout master
+       git submodules foreach git checkout master
+       git checkout -b stable_bump
+       git submodules foreach git checkout -b stable_bump
+
+-  Bump the version in each of the CMakeLists files in the :code:votca/votca repository and each of the submodules. This can be done by 
+   replacing :code:`<major>-dev` by :code:`<major>-rc.1` in :code:`votca/votca` and all submodules.
```

Comment:
Here I'm not sure what is replaced. Are those files or tags or something else?

Reply:
Better now?